install/kubernetes/quick-install.yaml

```diff
@@ -518,7 +567,7 @@ spec:
               key: wait-bpf-mount
               name: cilium-config
               optional: true
-        image: "docker.io/cilium/cilium:latest"
+        image: cilium/cilium:latest
```

Comment:
this is missing the registry `docker.io`

Reply:
it's implied in all container runtimes, similar patterns exist in almost every chart at https://github.com/helm/charts/tree/master/stable

Reply:
@seanmwinn yes, please add them back, thanks!

Reply:
done